Dockerfile

```
@@ -50,5 +50,16 @@ RUN set -ex; \
         echo "${TINI_SHA256_AMD64} */usr/bin/tini" | sha256sum -c; \
         chmod +x /usr/bin/tini; \
     fi
+
+
+# Async Profiler
+RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y tar gzip
+RUN mkdir /usr/local/async-profiler/ &&\
+      curl -L -o /usr/local/async-profiler/async-profiler.tar.gz https://github.com/async-profiler/async-profiler/releases/download/v3.0/async-profiler-3.0-linux-arm64.tar.gz &&\
```

I need something like 
```suggestion
      curl -L -o /usr/local/async-profiler/async-profiler.tar.gz https://github.com/async-profiler/async-profiler/releases/download/v3.0/async-profiler-3.0-linux-${TARGETARCH}.tar.gz &&\
```
to run locally (as I don't have arm64). Thought I do wonder about including MacOS support too.

---

I've tried initially running on macOS but as it uses an intermediate VM I wasn't able to. So I'd fail on macOS WDYT?

---

I think thats reasonable for now. 

One thing thats rolling around in my head is can we make async-profiler / flame graphs opt in and that way we can merge it with out having to iron out all the wrinkles.